src/transformers/models/xlnet/configuration_xlnet.py

```diff
@@ -106,12 +108,18 @@ class XLNetConfig(PretrainedConfig):
             Used in the SQuAD evaluation script.
         end_n_top (:obj:`int`, `optional`, defaults to 5):
             Used in the SQuAD evaluation script.
-        use_cache (:obj:`bool`, `optional`, defaults to :obj:`True`):
-            Whether or not the model should return the last pre-computed hidden states.
+        use_mems_eval (:obj:`bool`, `optional`, defaults to :obj:`True`):
+            Whether or not the model should make use of the recurrent memory mechanism in evaluation mode.
+        use_mems_train (:obj:`bool`, `optional`, defaults to :obj:`False`):
+            Whether or not the model should make use of the recurrent memory mechanism in train mode.
 
             .. note::
-                This flag behaves differently from with other models: it just controls the inference behavior, during
-                training the model always uses ``use_cache=True``.
+                For pre-training, it is recommended to set ``use_mems_train`` to :obj:`True`. For fine-tuning, it is
```

Comment:
```suggestion
                For pretraining, it is recommended to set ``use_mems_train`` to :obj:`True`. For fine-tuning, it is
```

Reply:
I think this happens quite often :D Grammarly tells me `pre-training` is better - but I'm 100% fine with changing my spelling to `pretraining`. I'll replace all pre-training's with pretrainings in `src/transformers` and `docs` :-)